javascript-create-module/template/dot/github/workflows/build.yml

```
@@ -0,0 +1,18 @@
+# Build and upload the package on every push
+name: Build
+
+on: push
+
+jobs:
+  build-and-upload:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+      - run: corepack enable
+      - run: yarn install --immutable
```

On CI env, the `--immutable` is not needed from what I recall
```suggestion
      - run: yarn install
```